lib/config/definitions.ts

```diff
@@ -1999,6 +1999,13 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'allowMergeWhenPipelineSucceeds',
```

Comment:
I'd prefer to have GitLab in the option name, so users don't have to read the description to know that. Maybe `useGitLabAutomerge` ? Cc @ViceIce @JamieMagee

Reply:
That sounds better, will update!

Reply:
The equivalent feature in Azure DevOps is called `azureAutoComplete`, so maybe `gitLabAutomerge`?

Reply:
In GitLab its actually called `merge when pipeline succeeds`, but that’s a bit lengthy, so I’ll go for `gitLabAutomerge` 😀

Reply:
Done with ea783946d0d76f953c6fe3b4c3f21026d2318010